src/core/modules/impl/HandlerExecutorFactory.ts

```diff
@@ -46,13 +49,28 @@ export class HandlerExecutorFactory implements ExecutorFactory<HandlerApi<unknow
     evaluationOptions: EvaluationOptions,
     warp: Warp
   ): Promise<HandlerApi<State>> {
+    let kvStorage = null;
+    if (evaluationOptions.useKVStorage) {
+      /*kvStorage = new Trie({
```

Comment:
the Trie version does not work (i.e. the integration test fails). Needs investigation.